service_capacity_modeling/tools/instance_families.py

```diff
@@ -0,0 +1,265 @@
+"""Contains definitions for AWS instance families."""
+
+HSW_IPC = 0.85
+SKX_IPC = 1.0
+ICX_IPC = SKX_IPC * 1.15  # harshad says 1.15, chatgpt says 1.18
+SPR_IPC = ICX_IPC * 1.12  # harshad says 1.12, chatgpt says 1.19
+ROME_IPC = 1.03
+MILAN_IPC = SKX_IPC * 1.15
+GENOA_IPC = MILAN_IPC * 1.13
+
+INSTANCE_TYPES = {
```

Comment:
I think this belongs in hardware, but we can leave it in tools if you prefer (or when we kinda figure out the shape of this data we like / can automate we could shift it then).